settings.py

```diff
@@ -19,6 +19,33 @@
     'debug_toolbar',
 )
 
+LOGGING['root'].update({'level': logging.DEBUG})
```

Comment:
yeah... I know this is what were (are!) doing in the conf/ files below but this isn't how dict updates work - 
before:
`LOGGING['root']` =  `{'handlers': ['mozlog'], 'level': logging.INFO}`
then running the `.update`
`LOGGING['root'].update({'level': logging.DEBUG})`
after
`LOGGING['root']` = `'root': {'level': logging.INFO}`
we lose the other dict elements because update isn't recursive.